diff --git a/node_modules/webpack/lib/ProgressPlugin.js b/node_modules/webpack/lib/ProgressPlugin.js
index 0afdfdd..7a57982 100644
--- a/node_modules/webpack/lib/ProgressPlugin.js
+++ b/node_modules/webpack/lib/ProgressPlugin.js
@@ -224,6 +224,7 @@ class ProgressPlugin {
 	 * @returns {void}
 	 */
 	_applyOnCompiler(compiler, handler) {
+		/*
 		compiler.hooks.validate.tap(PLUGIN_NAME, () => {
 			compiler.validate(
 				() => require("../schemas/plugins/ProgressPlugin.json"),
@@ -235,6 +236,7 @@ class ProgressPlugin {
 				(options) => require("../schemas/plugins/ProgressPlugin.check")(options)
 			);
 		});
+		*/
 
 		const showEntries = this.showEntries;
 		const showModules = this.showModules;
